src/main/java/tanks/gui/screen/ScreenOptionsGame.java

```diff
@@ -2,12 +2,14 @@
 
 import tanks.Drawing;
 import tanks.Game;
+import tanks.Panel;
 import tanks.gui.Button;
 
 public class ScreenOptionsGame extends Screen
 {
     public static final String autostartText = "Autostart: ";
     public static final String fullStatsText = "Stats animations: ";
+    public static final String pauseText = "Pause on defocus: ";

```

Comment:
IMHO, "unfocus" sounds better than "defocus"

Reply:
You can change it if you want, it's just that "Pause on lost focus" is too long for the normal button size